bluesky/traffic/route.py

```diff
@@ -48,7 +51,21 @@ def __init__(self):
         # default: False
         self.flag_landed_runway = False
 
-        return
+        self.iac = self.wpdirfrom = self.wpdistto = self.wpialt = \
+            self.wptoalt = self.wpxtoalt = None
+
+    @staticmethod
+    def get_available_name(data, name_, len_=2):
+        """

```

Comment:
Convenience method.  Currently, this is done twice in the code in different ways.  We should make a common definition.